superset/assets/src/explore/components/AdhocFilterEditPopoverSimpleTabContent.jsx

```diff
@@ -0,0 +1,248 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormGroup } from 'react-bootstrap';
+import VirtualizedSelect from 'react-virtualized-select';
+
+import AdhocFilter, { EXPRESSION_TYPES, CLAUSES } from '../AdhocFilter';
+import adhocMetricType from '../propTypes/adhocMetricType';
+import columnType from '../propTypes/columnType';
+import { t } from '../../locales';
+import {
+  OPERATORS,
+  TABLE_ONLY_OPERATORS,
+  DRUID_ONLY_OPERATORS,
+  HAVING_OPERATORS,
+  MULTI_OPERATORS,
+} from '../constants';
+import FilterDefinitionOption from './FilterDefinitionOption';
+import OnPasteSelect from '../../components/OnPasteSelect';
+import SelectControl from './controls/SelectControl';
+import VirtualizedRendererWrap from '../../components/VirtualizedRendererWrap';
+
+const $ = window.$ = require('jquery');
```

Comment:
is this `window` global necessary? some superset files do this but it depends on the libraries.

Reply:
good point let me try removing it